### PR TITLE
Features/assets

### PR DESCRIPTION
**Pull request by maggie97**

### CODE REVIEW

File: teckers/teckers/GoogleService-Info.plist [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (cortestavo) [inline teckers/teckers/GoogleService-Info.plist]: This file should be uploaded to repo, please remove it from PR

File: teckers/teckers/AppDelegate.swift [hunk 2]
```diff
@@ -7,18 +7,43 @@
...
+        FirebaseApp.configure()
+        
+        GIDSignIn.sharedInstance().clientID = FirebaseApp.app()?.options.clientID
+        GIDSignIn.sharedInstance().delegate = self
+        
         return true
     }
-
+    
+    func sign(_ signIn: GIDSignIn!, didSignInFor user: GIDGoogleUser!, withError error: Error?) {
```
Comment (cortestavo) [inline teckers/teckers/AppDelegate.swift]: Login a user is not a responsibility of the AppDelegate class, please move this logic to a different class

File: teckers/teckers/ViewController.swift [hunk 3]
```diff
@@ -7,12 +7,16 @@
...
+import GoogleSignIn
 
-class ViewController: UIViewController {
+class ViewController: UIViewController, GIDSignInUIDelegate {
 
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view.
+        GIDSignIn.sharedInstance().uiDelegate = self
+        //GIDSignIn.sharedInstance().signIn()
```
Comment (cortestavo) [inline teckers/teckers/ViewController.swift]: please try not to add commented code

File: teckers/teckers/AppDelegate.swift (hunk 2 above)
Comment (maggie97) [reply teckers/teckers/AppDelegate.swift]: I removed this

File: teckers/teckers/GoogleService-Info.plist (hunk 1 above)
Comment (maggie97) [reply teckers/teckers/GoogleService-Info.plist]: I removed this

File: teckers/teckers/AppDelegate.swift (hunk 2 above)
Comment (maggie97) [reply teckers/teckers/AppDelegate.swift]: I changed this function to view controller but I don't know if it's right

File: teckers/teckers/ViewController.swift (hunk 3 above)
Comment (maggie97) [reply teckers/teckers/ViewController.swift]: I removed this

File: teckers/teckers/AppDelegate.swift (hunk 2 above)
Comment (cortestavo) [reply teckers/teckers/AppDelegate.swift]: I feel it should be better to have an specific class to handle Google Sign in & Firebase interaction and to call methods os that class from a controller or the app delegate if necessary

**Pull request merged**